python-threatexchange/threatexchange/extensions/vpdq/README.md

```diff
@@ -0,0 +1,79 @@
+# Summary
+The description of vPDQ algorithm hashing and brute-force matching can be found in [readme](../../../../vpdq/README.md). This README describes the [FAISS](https://github.com/facebookresearch/faiss) matching process which is integrated with the vPDQ index in python-threatexchange.
+
+#### Faiss matching
+vPDQ features are a list of PDQ hashes embedded with Frame, Quality, and Timestamp(sec). When added to FAISS index, only PDQ hashes are indexed. Thus, extra information is stored in the wrapper index class to reformat the vPDQ feature when returning the results for searching. When we want to match a video (a list of vPDQ features) against the index, we also only input the PDQ hashes to search.
+
+For each query provided in queries, the returned sequence will contain a sequence of matches within the index
```

Comment:
nit: `For each query provided in queries..` is a bit confusing.
maybe something like "A query on the index is comprised of a sequence of individual PDQ hashes. Each of which will have a corresponding set of matches returned (or none in the case that specific hashes did not match anything in the index)."